lambda-resume-builder/src/Components/LogIn/LogIn.js

```diff
@@ -130,7 +143,8 @@ function SignIn(props) {
 const mapStateToProps = state => (
     
   {
-    login: state.login
+    login: state.login,
+    loggedIn: state.loggedIn
```

Comment:
## `loggedIn` isn't being used anywhere. It seems like you can probably remove it.

Reply:
I think it was you who made me create it!